project-base/tests/FrontendApiBundle/Functional/Payment/PaymentsTest.php

```diff
@@ -46,8 +58,8 @@ public function testPayments(): void
                             'vatAmount' => '0.00',
                         ],
                         'images' => [
-                            ['url' => 'http://webserver:8080/content-test/images/payment/default/53.jpg'],
-                            ['url' => 'http://webserver:8080/content-test/images/payment/original/53.jpg'],
+                            ['url' => $this->overwriteDomainUrl . '/content-test/images/payment/default/53.jpg'],
```

Comment:
In `GraphQlTestCase` there could exist function like `getFullUrlPath(string $uri)` for getting this.

Reply:
Fixed